specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/examples/createBasicAKSCompute.json

```diff
@@ -13,18 +13,6 @@
         }
     },
     "responses": {
-        "200": {
```

Comment:
Why is this removed? Same questions for other such removals of 200 responses

Reply:
Because if you look at our code (the MLC microservice code) and our internal swagger, we don't return a 200 anymore. We return 201 always for correct calls.
And the swagger needs to represent the rp correctly.